specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,689 @@
+        "modelAsString": true
+      }
+    },
+    "DeploymentType": {
+      "type": "object",
+      "description": "Object representing DeploymentType for Managed CCF.",
+      "title": "DeploymentType",
+      "properties": {
+        "languageRuntime": {
+          "readOnly": false,
+          "description": "Unique name for the Managed CCF.",
+          "$ref": "#/definitions/LanguageRuntime"
+        },
+        "appSourceUri": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Source Uri containing ManagedCCF code"
+        }
+      }
+    },
+    "CertificateTags": {
+      "description": "Tags for Managed CCF Certificates",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Additional tags for Managed CCF Certificates",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "MemberIdentityCertificate": {
+      "type": "object",
+      "description": "Object representing MemberIdentityCertificate for Managed CCF.",
+      "title": "MemberIdentityCertificate",
+      "properties": {
+        "certificate": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Member Identity Certificate"
+        },
+        "encryptionkey": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Member Identity Certificate Encryption Key"
+        },
+        "tags": {
+          "items": {
+            "$ref": "#/definitions/CertificateTags"
+          }
+        }
+      }
+    },
+    "NodeCount": {
+      "type": "integer",
+      "format": "int32",
+      "description": "Number of CCF nodes in the Managed CCF.",
+      "title": "NodeCount",
+      "default": 3
+    },
+    "ManagedCCFProperties": {
+      "type": "object",
+      "description": "Additional Managed CCF properties.",
+      "title": "ManagedCCFProperties",
+      "properties": {
+        "appName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique name for the Managed CCF."
+        },
+        "appUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for calling Managed CCF Service."
+        },
+        "identityServiceUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for accessing network identity."
+        },
+        "memberIdentityCertificates": {
+          "type": "array",
+          "description": "List of member identity certificates for  Managed CCF",
+          "items": {
+            "$ref": "#/definitions/MemberIdentityCertificate"
+          },
+          "x-ms-identifiers": []
+        },
+        "deploymentType": {
+          "description": "Deployment Type of Managed CCF",
+          "$ref": "#/definitions/DeploymentType"
+        },
+        "runningState": {
```

Comment:
>runningState [](http://example.com/codeflow?start=9&length=12)

+readonly?

Reply:
The `runningState` property for the managedccf resource has the same use as the `runningState` property for the confidentialledger resource [found here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-01-26-preview/confidentialledger.json#L593) and should not be readonly.